static/js/services/enketo.js

```diff
@@ -21,6 +22,7 @@ angular.module('inboxServices').service('Enketo',
     var objUrls = [];
     var xmlCache = {};
     var FORM_ATTACHMENT_NAME = 'xml';
+    var REPORT_ATTACHMENT_NAME = this.REPORT_ATTACHMENT_NAME = 'content';
```

Comment:
Is this double-assignment is to allow external access while avoiding `this.` references internally (where they'll probably be confusing anyway)?